docs/static/java-filter.asciidoc

```diff
@@ -212,6 +218,22 @@ public String getId() {
 For filter plugins, the `getId` method should always return the id that was provided to the plugin through its
 constructor at instantiation time.
 
+[float]
+==== close method
+
+[source,java]
+-----
+@Override
+public void close() {
+    this.sourceField = null;
+    return;
+}
+-----
+
+After a pipeline has been shut down, its filters are closed.
+If your plugin holds additional resources such as database connections,
+implementing this method will allow you to free up those resources.
+
```

Comment:
@karenzone  May I ask your review on this please ?

Reply:
What about?

When using filter plugins in a pipeline, plugins can require additional resources, such as database connections. When a pipeline is shutdown and filters are closed, this `close` method will allow the plugin to free up those resources.

Or maybe:


Filter plugins can use additional resources to perform operations, such as creating new database connections. Implementing the `close` method will allow the plugins to free up those resources when shutting down the pipeline.

Reply:
Fixed with ea5008d41df7a00af8a344d6a5161ad64ab8231f